torch/optim/lr_scheduler.py

```diff
@@ -801,8 +810,9 @@ def _get_closed_form_lr(self):
 
 
 class CosineAnnealingLR(LRScheduler):
-    r"""Set the learning rate of each parameter group using a cosine annealing
-    schedule, where :math:`\eta_{max}` is set to the initial lr and
+    r"""Set the learning rate of each parameter group using a cosine annealing schedule.
+
+    Where :math:`\eta_{max}` is set to the initial lr and
```

Comment:
```suggestion
    :math:`\eta_{max}` is set to the initial lr and
```
Combine with following line